### PR TITLE
Added Disney International HD

### PR DESCRIPTION
**Pull request by Nintendocustom**
#1256 
It isn't really International.

### CODE REVIEW

**Review by freearhey — CHANGES_REQUESTED** [review]
If I got it right, it's not an Indian channel either: https://www.lyngsat.com/tvchannels/us/Disney-International-HD.html

**Comment by freearhey** [comment]
I mean, it's being broadcast from the US. At least that's what the lyngsat description says.

**Comment by Nintendocustom** [comment]
On Wikipedia it's India: https://en.wikipedia.org/wiki/Disney_International_HD.

File: channels/int.m3u [hunk 1]
```diff
@@ -33,6 +33,8 @@ https://video.sekindo.com/blocktv/ch1/tracks-v3a1/mono.m3u8
 https://live.creacast.com/cna/smil:cna.smil/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/tXTjWgP.png" group-title="Religious",Christian Youth Channel
 http://media.smc-host.com:1935/cycnow.com/cyc2/chunklist_w258510680.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/7/7f/Disney_International_HD_logo.png/220px-Disney_International_HD_logo.png" tvg-country="IN" tvg-url="" group-title="Kids",Disney International HD
+http://stream-7b694ce7e.nmbs.top/m3u/Kids/10166da03f.m3u8
```
Comment (sultanfendonus) [inline channels/int.m3u:37]: The channel is broken. You should review it.

**Review by sultanfendonus — CHANGES_REQUESTED** [review]

File: channels/int.m3u (hunk 1 above)
Comment (freearhey) [reply channels/int.m3u:37]: @sultanfendonus for me it's working fine
Captura de pantalla 2020-05-19 a las 14 55 51

**Review by freearhey — APPROVED** [review]

**Pull request merged**